### DISCUSSION TITLE
Listing Detail API

### DISCUSSION THREAD

**Discussion by sanandrea** [AnthonyBloomer/daftlistings#158] [General]
Is there a public listing API from Daft? What I am looking for are more photos from the listing. I might be wrong, but it seems that the ones returned from the search API do not contain the entire photos.

**Comment by sdfordham** [comment]
Daft has a paid for API, this library just is based on the one used by the frontend. Can you give an example of what you mean about not containing the entire photos?

**Reply by sanandrea** [reply]
I think the search API returns the photos used in the thumbnail and maybe 3-4 subsequent ones (also thumbnail size). Anyway I do not need the entire list of photos anymore.
